src/main/java/ru/practicum/shareit/booking/BookingController.java

```diff
@@ -22,46 +27,50 @@ public class BookingController {
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
-    public BookingResponseDto createBooking(@RequestHeader(REQUEST_HEADER_USER_ID) Long bookerId,
-                                            @RequestBody final BookingRequestDto requestDto) {
+    public BookingOutputDTO createBooking(@RequestHeader(REQUEST_HEADER_USER_ID) Long bookerId,
```

Comment:
Необходимо добавить проверку на то, что передаваемый ID элемента (itemId) действительно существует в базе данных, прежде чем позволить создать бронирование. Это предотвратит создание бронирований на несуществующие элементы.

Reply:
Добрый день.
Данная проверка уже присутствует.
На уровне сервисного слоя при создании бронирования, проверка пользователя и предмета, через вызов приватных методов.
```suggestion
    @Override
    @Transactional
    public BookingOutputDTO create(BookingInputDTO inputDTO) {

        UserOutputDTO booker = validateUserById(inputDTO.getBookerId());
        ItemShortOutputDTO item = validateItemById(inputDTO.getItemId());
```